src/classes/Task/Task.php

```diff
@@ -16,12 +16,15 @@ class Task
     public const ACTION_DONE = 'done';
     public const ACTION_DENY = 'deny';
 
-    private $customer_id;
-    private $implementer_id;
-    private $status;
+    private int $customer_id;
+    private int $implementer_id;
+    private string $status;
 
     public function __construct($customer_id, $status)
     {
+        if (!in_array($status, $this->getMapStatus)) {
```

Comment:
if (!in_array($status, array_keys($this->getMapStatus()))) {
            throw new StatusException("$status doesn't exist!");
        }